README.md

```diff
@@ -1,15 +1,17 @@
 # WEB3DEV NFT Game
 
 ### **Bem-vind@ 👋**
+
 Para iniciar o projeto, clone esse repo e siga esses comandos:
 
 1. Rode `npm install` na raíz do seu projeto
 2. Rode `npm run start` para iniciar
 3. Comece a codar!
 
 ### O que é a pasta `.vscode`?
-Se você usa VSCode, incluímos uma lista de extensões sugeridas que vão ajudar nesse projeto! Assim que abrir esse projeto, verá um popup perguntando se deseja instalar essas extenções :).
```

Comment:
why this change?

Reply:
following the [CommunMark](https://xiangxing98.github.io/Markdownlint_Rules.html#md022---headings-should-be-surrounded-by-blank-lines) which defines that headers should be surrounded by blank lines